internal/analytics/deferred.go

```diff
@@ -34,10 +34,18 @@ func deferrerFilePath(cfg Configurable) string {
 func isDeferralDayAgo(cfg Configurable) bool {
 	df := deferrerFilePath(cfg)
 	stat, err := os.Stat(df)
+	if os.IsNotExist(err) {
+		tErr := fileutils.Touch(deferrerFilePath(cfg))
+		if tErr != nil {
+			logging.Errorf("Failed to create deferrer time stamp file: %v", tErr)
+		}
+		return false
+	}
 	if err != nil {
-		logging.Errorf("Could not stat file: %s, error: %v", df, err)
+		logging.Debug("Could not stat deferrer file: %s, error: %v", df, err)
```

Comment:
With the fix we have the issue at hand should be addressed. I'd still want to hear about these errors if they occur for some other reason.